build/gulp/ts.js

```diff
@@ -59,12 +59,6 @@ gulp.task('ts-copy-bundle', gulp.series(
             .pipe(replace(/(interface JQuery\b[\s\S]*?{)[\s\S]+?(})/gm, '$1$2'))
             .pipe(gulp.dest(packageBundlesPath));
     },
-
-    function writeAngularHack() {
```

Comment:
Looks like we shouldnt remove this function. It's related to the devextere-angular not native angular